frontend/src/features/dashboard/components/DeliveryInformationModal.tsx

```diff
@@ -34,30 +46,553 @@ const GridItem = ({ label, value }: { label: string; value: string }) => (
+                        justifyContent="center"
+                        width="100%"
+                        height="100%"
+                    >
+                        {data.row.customerOrderNumber ? (
+                            <>
+                                <Box>{`${data.row.customerOrderNumber} / ${data.row.customerOrderPositionNumber}`}</Box>
+                                <Box>{data.row.supplierOrderNumber}</Box>
+                            </>
+                        ) : (
+                            '-'
+                        )}
+                    </Box>
+                );
+            },
+        },
+        {
+            field: 'trackingNumber',
+            headerName: 'Tracking Number',
+            headerAlign: 'center',
+            width: 130,
+            renderCell: (data: { row: Delivery }) => {
+                return (
+                    <Box display="flex" textAlign="center" alignItems="center" justifyContent="center" width="100%" height="100%">
+                        {data.row.trackingNumber}
+                    </Box>
+                );
+            },
+        },
+        {
+            field: 'incoterm',
+            headerName: 'Incoterm',
+            headerAlign: 'center',
+            width: 150,
+            renderCell: (data: { row: Delivery }) => {
+                return (
+                    <Box
+                        display="flex"
+                        flexDirection="column"
+                        textAlign="center"
+                        alignItems="center"
+                        justifyContent="center"
+                        width="100%"
+                        height="100%"
+                    >
+                        <Box>{INCOTERMS.find((i) => i.key === data.row.incoterm)?.value ?? '-'}</Box>
+                        <Box>({data.row.incoterm})</Box>
+                    </Box>
+                );
+            },
+        },
+        {
+            field: 'delete',
+            headerName: '',
+            sortable: false,
+            disableColumnMenu: true,
+            headerAlign: 'center',
+            type: 'string',
+            width: 30,
+            renderCell: (data: { row: Delivery }) => {
+                return (
+                    <Box display="flex" textAlign="center" alignItems="center" justifyContent="center" width="100%" height="100%">
+                        {!data.row.reported && <Button variant="text" color="error" onClick={() => handleDelete(data.row)}>
+                            <Delete></Delete>
+                        </Button>}
+                    </Box>
+                );
+            },
+        },
+    ] as const;
+};
+
+const isValidDelivery = (delivery: Partial<Delivery>) =>
+    delivery.ownMaterialNumber &&
+    delivery.originBpns &&
+    delivery.partnerBpnl &&
+    delivery.destinationBpns &&
+    delivery.quantity &&
+    delivery.measurementUnit &&
+    delivery.dateOfDeparture &&
+    delivery.dateOfArrival &&
```

Comment:
Please add validations for actual < now

Also checked that worked in modal but failed in backend:
- arrival actual and departure estimated
- departure < arrival

Reply:
![image](https://github.com/eclipse-tractusx/puris/assets/116013795/818d6a5a-5cd2-4a2a-a265-caf9867a320a)
Can we make the highlight on the arrival type for the actual / estimated issue?